src/JuliaRunClient.jl

```diff
@@ -301,6 +301,16 @@ function _type_name_query(ctx::Context, path::String, job::JRunClientJob, query:
     parse_resp(resp)
 end
 
+function initialize(num_workers, ctx=Context(), job=self())
+    initParallel()
+    setJobScale(ctx, job, num_workers)
+    waitForWorkers(num_workers)
+end
+
+function release(ctx=Context(), job=self())
```

Comment:
So will `Context()` always return the current context? Or will it create a new one?

Reply:
Will return the current context